seaborn/tests/test_matrix.py

```diff
@@ -245,6 +245,14 @@ def test_heatmap_annotation_with_mask(self):
         for val, text in zip(df_masked[::-1].compressed(), ax.texts):
             nt.assert_equal("{:.1f}".format(val), text.get_text())
 
+    def test_heatmap_annotation_mesh_colors(self):
+
+        ax = mat.heatmap(self.df_norm, annot=True)
+        mesh = ax.collections[0]
+        nt.assert_equal(len(mesh.get_facecolors()), 4 * 8)
```

Comment:
I would change this to infer the needed size from the dataframe itself (i.e. `self.df_norm.size`). As written, this test would break (possibly in a confusing way) if `df_norm` is changed. Otherwise looks great.